### DISCUSSION TITLE
[REQ] ...what about categories/subcategories/wizard ?

### DISCUSSION THREAD

**Discussion by MarcoRavich** [webprofusion/OpenAudio#130] [General]
Hi there, very nice project folks !

I've added [HyMPS](https://github.com/forart/HyMPS#readme) in collections, please check it out both for resources and categorizations (if you need).

It would be cool to collaborate for evolve each other (I would like to better organize it, with automations too)...

...last but not least: how about a web wizard to allow an easy resource submission (getting users to pre-order them) ?

Hope that inspires !

**Comment by webprofusion-chrisc** [comment]
Yes we'd love to accept more contributions. For an automated form I think we'd probably head towards a database (that gets exported as json so others can still use the data from here). As github doesn't host functions I'd probably lean towards Cloudflare workers with either Key/Value storage or D1 (sqlite), then some sort of periodic export to json into the github repo. Once you have a database you then need administrative functions (edit, delete etc).

For organization I think attaching tags (multiple categories, other metadata like OSS license type, framework etc),  would then let the user slice the info in whatever way they want.
